internal/controller/prefix_controller.go

```diff
@@ -132,9 +132,19 @@ func (r *PrefixReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 			return ctrl.Result{}, err
 		}
 
+		if prefixClaim.Status.ParentPrefix == "" {
```

Comment:
I am not sure if this is required, but I would like to log this in case we change more code in the future and this condition gets tripped somehow. What do you think?

Reply:
https://github.com/netbox-community/netbox-operator/pull/90#discussion_r1796932930